### ISSUE TITLE
Virtual scroll must have activated and deactivated states.

### ISSUE THREAD

**Issue by yahao87** [quasarframework/quasar#12981]
### What happened?

If you scroll slightly from the top of the screen while virtual scroll is deactivated keep-alive, the scrolling behaves strangely.
I found it in Quasar v1 and so is v2.

### What did you expect to happen?

When deactivated, there should be no events.

### Reproduction URL

https://codesandbox.io/s/quasar-virtual-scroll-bug-1owlfk?file=/src/pages/Index.vue

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Components (quasar)

### Additional context

https://github.com/quasarframework/quasar/blob/0acf0de6fa6fc0d3e548db73e7cd2f0a4fa5d3ff/ui/src/components/virtual-scroll/QVirtualScroll.js#L112-L122

The code below should be added.
```
 activated () {
    this.__configureScrollTarget()
  },

  deactivated () {
    this.__unconfigureScrollTarget()
  },
```

**Comment by pdanpdan** [comment]
Very good catch, thank you.

**Comment by rstoenescu** [comment]
Fix will be available in v2.6.3 and v1.18.8
